content/posts/02_staff_expectations/index.md

```diff
@@ -0,0 +1,116 @@
+positions. Career progression broadens the scope of work andresponsibilities. The environment and problems get more ambiguous. It's not easy
+to define a clear and measurable set of expectations, because staff roles also differ between companies and
+their actual needs. They can also vary between different [staff engineer archetypes](https://staffeng.com/guides/staff-archetypes).
+
+In the article, we will focus mostly on the **Tech Lead** archetype, which is the most popular in the industry. We will present some 
+expectations for staff engineers based on our experience. The list is not complete and there are many very important skills and 
+traits a staff engineer should have. We wanted to describe those that are fundamental regardless of the specific situation.
+
+# So what are those expectations?
+
+## Get things done
+![](./done.jpg)
+
+This might as well be applied to the earlier positions. But for staff engineers, it gets even more important. Staff engineers have to get 
+their own work done and make sure others can do it too. They should help other engineers and teams manage and prioritize complex projects. 
+They should act as a "glue" to ensure engineering teams are working and delivering smoothly. They should observe and eliminates the developer
+experience problems and increase productivity.
+
+Some of the examples of getting stuff done:
+* Help the team with project management and prioritization.
+* Identify hidden risks and dependencies, which could disturb project delivery.
+* Advocate with leadership and find the necessary resource to finish a project.
+* Find a way to solve a particularly difficult problem.
+
+## Be a role model and mentor others
+![](./arrows.png)
+
+At some point in a career, you usually start to have a bigger impact through others, rather than by your individual contributions. This 
+happens not only on the management track but also for technical leaders. Some foretaste comes even for senior engineers, but you get a full
+experience of it in staff+ positions. 
+
+Younger people start to look up to staff engineers and even try to mimic their behavior. Staff engineers become role models for
+others. That's why it's important to demonstrate the company's values and culture through actions. They get the leverage to align people and 
+multiply the outcome.
+
+Staff engineers also get an opportunity (and obligation) to 
+[mentor](https://blog.pragmaticengineer.com/developers-mentoring-other-developers/) other people. We consider it a great privilege, to be 
+able to pass your experiences and your knowledge to other people. Helping people grow and succeed is very important. Not only you're 
+actively using your time for others, but also you're increasing the area of impact. We also believe, that during mentoring, you can learn 
+a tremendous amount of things about yourself, efficient communication, setting goals, etc.
```

Comment:
This last sentence is imho hanging here in a very vague way. What does it bring to the table?

Reply:
You mean this one?
> We also believe, that during mentoring, you can learn 
a tremendous amount of things about yourself, efficient communication, setting goals, etc.

I thought it's clear. During mentorship, a mentors can also upskill themselves. When you're supposed to explain and teach something, you need a thorough understanding of those concepts. I used ChatGPT here, have a look if it looks better now.